include/deal.II/physics/elasticity/kinematics.h

```diff
@@ -223,7 +223,7 @@ namespace Physics
        * gradient).
        * The result is expressed as
        * @f[
-       *  \mathbf{d} := \frac{1}{2} \left[ \mathbf{l} + \mathbf{l}^{T} \right] \, .
```

Comment:
This is what's being deleted. There shouldn't be a "." there, and to be consistent with the other formulae I'd prefer not to put in a ",".